modules/meeting/app/controllers/meetings_controller.rb

```diff
@@ -127,15 +128,6 @@ def build_meeting
     @meeting.author = User.current
   end
 
-  def find_optional_project
-    return true unless params[:project_id]
-
-    @project = Project.find(params[:project_id])
-    authorize
-  rescue ActiveRecord::RecordNotFound
-    render_404
-  end
-
```

Comment:
Always good to get rid of those controller specific overrides if they are not needed. 🙇